app/controllers/api/v1/projects_controller.rb

```diff
@@ -22,11 +23,15 @@ def project_params
     params.permit(:name, :vanity_url, :homepage_url, :repo_url, :coverity_project_id, :license_name)
   end
 
+  # rubocop:disable Metrics/AbcSize
   def save_project
     populate_project_from_forge(project_params[:repo_url], true)
+    @project.name = params[:name] if @project && params[:name]
+    @project.coverity_project_id = params[:coverity_project_id] if @project && params[:coverity_project_id]
     create_params(@project) if @project
     @project = ProjectBuilder.new(current_user, @project_params || {}).fetch_project
   end
```

Comment:
You can change the logic to the following which is the same behaviour. It will get rid of the metrics/abcsize warning:
```
    populate_project_from_forge(project_params[:repo_url], true)
    if @project
      @project.name = params[:name]
      @project.coverity_project_id = params[:coverity_project_id]
      create_params(@project)
    end
    ProjectBuilder.new(current_user, @project_params || {}).fetch_project
```

Also when u pass `project` as mentioned in the #1654 PR, stop using `@project` here and pass `project` back to calling method instead.